packages/block-editor/src/components/rich-text/index.js

```diff
@@ -186,12 +191,16 @@ export function RichTextWrapper(
 					args: relatedBinding.args,
 				} );
 
+			const bindingKey =
+				fieldsList?.[ relatedBinding?.args?.key ]?.label ??
+				blockBindingsSource?.label;
+
 			const _bindingsPlaceholder = _disableBoundBlock
-				? relatedBinding?.args?.key || blockBindingsSource?.label
+				? bindingKey
 				: sprintf(
 						/* translators: %s: source label or key */
```

Comment:
This comment for translators needs to get updated to reflect changes.

Reply:
Updated in the latest commit.